.github/workflows/docs.yaml

```diff
@@ -0,0 +1,61 @@
+name: Documentation
+# FOLLOWING https://coderefinery.github.io/documentation/gh_workflow/
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [closed] # Run conditionals based on opened/merged PRs
+    paths:
+      - 'docs/*'  # Trigger the workflow only when changes are made in the 'docs' directory
+    branches:
+      - 'main' # trigger publish only on closed PR in main
+  push:
+    paths:
+      - 'docs/**'  # Trigger the workflow only when changes are made in the 'docs' directory
+    branches:
+      - '**' # run doc build every time docs change in remote branch
+
+
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      if: ${{ github.event_name == 'push'}}
```

Comment:
The workflow logic I have currently for that file is:

1. Check to see if the docs directory has changes, based on a push from any branch. 
2. If there is a push to a branch and `docs` has changed, run only the steps up to "Deploy to GitHub pages"
3. Otherwise, if those checks complete successfully and that branch makes it to a PR, and that PR gets merged, build the documentation. 
4. 
5. So, the assumption is that since we ran all those we don't need to run them again on merge, but we could if we feel it's easier.